allensdk/brain_observatory/ophys/trace_extraction/__init__.py

```diff
@@ -0,0 +1,8 @@
+import warnings
+
+warnings.warn("trace_extraction functionality has been moved from AllenSDK "
+              "to https://github.com/AllenInstitute/ophys_etl_pipelines ."
+              "The functionality in this AllenSDK package will be removed "
```

Comment:
What do we mean though by `The functionality`?

Is it just this subpackage, or are we also planning to remove stuff like:

https://github.com/AllenInstitute/AllenSDK/blob/8853fc05d5798174ff833e148d247d538bac9fb0/allensdk/brain_observatory/ophys/trace_extraction/__main__.py#L14

Which is doing most of the heavy lifting for the trace extraction subpackage?